available/README.md

```diff
@@ -0,0 +1,51 @@
+# Hooks
+
+The hooks in this directory are ready to symlinked to whichever hook points you wish to use them at.
+
+## `reload_`
+
+Reload a specific service.  To use, create a symlink to it with the name `reload_{service}` with the name of the service you wish to reload from the appropriate hook directory.  For example, to reload postfix create a symlink to `reload_` with the name `reload_postfix`.  Combine with `ifdomain_` (see below) to reload services only when the SSL certficiate they use is updated.
+
+## `ifdomain_`
+
+Only runs the hook if the domain triggering it matches.
+
+Due to the restrictions of run-parts (only letters, numbers, dashes and underscores are permitted), domains in the filenames needs to have the periods replaced with dashes.  For example, 'google.co.uk' would be 'google-co-uk' in the filename.
+
+Use it by creating a symlink with the name `ifdomain_{domain}_{hook to run}`.  The hook you want to run must exist in the same directory as the `ifdomain_` script your symlink points to.  For example, `ifdomain_google-co-uk_reload_nginx` will call the `reload_nginx` hook if the domain is 'google.co.uk'.  (Note that this means you need to create a symlink, in the same place as `ifdomain_` is, called `reload_nginx` pointing to `reload_`, if you want to use that).
```

Comment:
If I'm reading this correctly, the symlink would need to be in the `available` directory, unless the user has rearranged things?   Might be worth clarifying that, or perhaps list the necessary links for an example, like this:

```
deploy-cert/ifdomain_example-com_reload_nginx => ../available/ifdomain_
available/reload_nginx => reload_
```

Reply:
Firstly, thank you for the thorough review I appreciate your time and effort.

Please can you review the wording in 8b0c67c to see if that clarifies it?  I added a simpler example and then the more complex chained-chain hook example, with the list of hook scripts and links as you suggested.

Hope that meets with your approval?

Thanks again,

Laurence